src/Authentication/Authentication/test/loadEnv.ps1

```diff
@@ -0,0 +1,29 @@
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+$envFile = 'localEnv.json'
+if (Test-Path -Path (Join-Path $PSScriptRoot $envFile)) {
+    $envFilePath = Join-Path $PSScriptRoot $envFile
+} else {
+    $envFilePath = Join-Path $PSScriptRoot '..\$envFile'
+}
+$env = @{}
+if (Test-Path -Path $envFilePath) {
+    $env = Get-Content (Join-Path $PSScriptRoot $envFile) | ConvertFrom-Json
+} else {
+    $env.TenantIdentifier = ${env:TENANTIDENTIFIER}
+    $env.ClientIdentifier = ${env:CLIENTIDENTIFIER}
+    $env.CertificateThumbprint = ${env:CERTIFICATETHUMBPRINT}
+}
+$PSDefaultParameterValues=@{"Connect-Graph:TenantId"=$env.TenantIdentifier; "Connect-Graph:ClientId"=$env.ClientIdentifier; "Connect-Graph:CertificateThumbprint"=$env.CertificateThumbprint}
```

Comment:
```suggestion
$PSDefaultParameterValues=@{"Connect-MgGraph:TenantId"=$env.TenantIdentifier; "Connect-MgGraph:ClientId"=$env.ClientIdentifier; "Connect-MgGraph:CertificateThumbprint"=$env.CertificateThumbprint}
```
nit: Let's avoid using aliases where possible since they could change.